gwt-uploader/src/main/java/org/wisepersist/gwt/uploader/client/Uploader.java

```diff
@@ -297,51 +226,27 @@ public void setHttpHeaders(JSONObject httpHeaders) {
   private String ajaxUploadURL;
 
   /**
-   * By default both the XMLHttpRequest Level 2/Ajax uploads and the SWFUpload/Flash uploads will
+   * By default both the XMLHttpRequest Level 2/Ajax uploads will
    * be posted to whatever URL is specified via the {@link #setUploadURL(String)} method.
-   * However, if you'd like the Ajax uploads to be posted to a different URL than the SWF
-   * uploads, you can call this method to set a unique URL for the Ajax uploads (and then
-   * only the SWF uploads will use the general "uploadURL").
+   * However, if you'd like the Ajax uploads to be posted to a different URL, you can call 
+   * this method to set a unique URL for the Ajax uploads.
    *
    * @param ajaxUploadURL The URL that only the Ajax uploads should be posted to (or null
-   *                      to uploader both the Ajax and SWF uploads to whatever
+   *                      to uploader the Ajax uploads to whatever
    *                      the {@link #setUploadURL(String)} option has been set to.)
    * @return A reference to this {@link Uploader} instance for convenient method chaining.
    */
   public Uploader setAjaxUploadURL(String ajaxUploadURL) {
     this.ajaxUploadURL = ajaxUploadURL;
-    return this.setOption("/ajax_upload_url", uploadURL);
-  }
-
-  private boolean ajaxUploadEnabled = true;
-
-  /**
-   * By default the component will attempt to use a DOM based Ajax uploader process if it detects
-   * that the browser can support the necessary requirements of the "XMLHttpRequest Level 2".  And
-   * then, only if the browser can not support the "XMLHttpRequest Level 2" mechanism will the
-   * component failover to using the Flash based SWFUpload mechanism instead.  However,
-   * if you wish to only use the SWFUpload mechanism, then this option can be disabled.
-   *
-   * @param ajaxUploadEnabled By default ajax uploader is enabled and will be used as long as the
-   *                          browser supports the "XMLHttpRequest Level 2" object.  Setting this
-   *                          option to "false" will instead force the component to avoid using
-   *                          the Ajax approach and instead utilize the SWFUpload/Flash technique
-   *                          only.
-   * @return A reference to this {@link Uploader} instance for convenient method chaining.
-   */
-  public Uploader setAjaxUploadEnabled(boolean ajaxUploadEnabled) {
-    this.ajaxUploadEnabled = ajaxUploadEnabled;
     return this;
   }
 
+  private boolean ajaxUploadEnabled = true;
```

Comment:
If we only have Ajax upload, do we still need this variable at all?

Reply:
Yes your are right - removed it.